src/main/java/com/cx/restclient/CxSASTClient.java

```diff
@@ -379,6 +383,40 @@ private long createRemoteSourceScan(long projectId) throws IOException {
 
         }
     }
+    
+    public String getContentTypeAndApiVersion(CxScanConfig config, String apiName) {
+    	   CxVersion cxVersion = config.getCxVersion();
+    	   String sastVersion = cxVersion.getVersion();
+    	   String apiVersion = CONTENT_TYPE_APPLICATION_JSON_V1;
+
+    	   if (sastVersion != null && !sastVersion.isEmpty()) {
+    	       String[] versionComponents = sastVersion.split("\\.");
+
+    	       if (versionComponents.length >= 2) {
+    	           String currentVersion = versionComponents[0] + "." + versionComponents[1];
+    	           float currentVersionFloat = Float.parseFloat(currentVersion);
+
+    	           if (currentVersionFloat >= 9.4) {
+    	               if ("SAST_RETENTION_RATE".equals(apiName) && config.isEnableDataRetention()) {
```

Comment:
Use equalIgnoreCase at all places and use consistent naming convention for 'apiName'. In one case it is all upper case with underscore while at other place it is camel casing.